crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -70,27 +80,38 @@ def restore_chef
         true
       end
 
+      def restore_files(source, destination)
+        # keep the permissions of the files that are already in place
+        src_path = @data.join("crowbar", source)
+        dest_is_dir = system("sudo", "-i", "test", "-d", destination)
```

Comment:
IIRC ```destination``` is an absolute path here. But I guess you're still right the ```-i``` shouldn't be needed here. (And in the other place)